datamodel/src/test/kotlin/com/hartwig/actin/datamodel/molecular/TestMolecularFactory.kt

```diff
@@ -293,6 +293,38 @@ object TestMolecularFactory {
         isAssociatedWithDrugResistance = null
     )
 
+    fun createMinimalVariant() = Variant(
+        chromosome = "",
+        position = 0,
+        ref = "",
+        alt = "",
+        type = VariantType.UNDEFINED,
+        variantAlleleFrequency = null,
+        canonicalImpact = createMinimalTranscriptImpact(),
+        otherImpacts = emptySet(),
+        extendedVariantDetails = null,
+        isHotspot = false,
+        isReportable = false,
+        event = "",
+        driverLikelihood = null,
+        evidence = TestClinicalEvidenceFactory.createEmpty(),
+        gene = "",
+        geneRole = GeneRole.UNKNOWN,
+        proteinEffect = ProteinEffect.UNKNOWN,
+        isAssociatedWithDrugResistance = false
+    )
+
+    fun createMinimalTranscriptImpact() = TranscriptVariantImpact(
```

Comment:
Would stick to ordering as in datamodel. Also, `isSpliceRegion` can be null in minimal version